README.md

```diff
@@ -301,14 +421,18 @@ $ smart-diff                    # check diff relative to parent branch
     ... git push, etc.
 ```
 
-#### GoLand configuration
+</details>
+
+#### GoLand Configuration
+
+<details><summary>Click to expand</summary>
 
 If you're using GoLand for development, the following can help improve the experience.
 
-Make sure the `Protocol Buffers` plugin is installed. The plugin comes installed by default in GoLand.  
-If it isn't, use `Help | Find Action...`, type `Plugins` and hit enter, then switch to `Marketplace`, type its name and install the plugin.  
+Make sure `Protocol Buffer Editor` plugin is installed. If it isn't, use `Help | Find Action...`, type `Plugins` and hit
+enter, then switch to `Marketplace`, type its name and install the plugin.  
```

Comment:
I think the info on the main branch is newer, as I remember updating the GoLand setup, because the plugin got renamed and now comes standard with all GoLand installs :D 

```suggestion
Make sure the `Protocol Buffers` plugin is installed. The plugin comes installed by default in GoLand.  
If it isn't, use `Help | Find Action...`, type `Plugins` and hit enter, then switch to `Marketplace`, type its name and install the plugin. 
```